ocs_ci/utility/uninstall_openshift_logging.py

```diff
@@ -0,0 +1,52 @@
+"""
+Function to teardown the openshift-logging
+"""
+import logging
+
+from ocs_ci.ocs import constants
+from ocs_ci.ocs import ocp
+
+logger = logging.getLogger(__name__)
+
+
+def uninstall_cluster_logging():
+    """
+    Function to uninstall cluster-logging from the cluster
+    1. Deletes the project "openshift-logging"
+        & "openshift-operators-redhat"
+    2. Deletes the CSC resource "cluster-logging-operator"
+        & "elasticsearch"
+    """
+    namespace_obj1 = ocp.OCP(
+        kind='Namespaces', namespace='openshift-logging'
+    )
+    namespace_obj2 = ocp.OCP(
+        kind='Namespaces', namespace='openshift-operators-redhat'
+    )
+
+    if namespace_obj1.get():
+        logger.info("The namespaces openshift-logging exists, Deleting.......")
+        assert namespace_obj1.delete(resource_name='openshift-logging')
+        logger.info("The project openshift-logging got deleted successfully")
+    else:
+        logger.error("The project openshift-logging does not exists")
+
+    if namespace_obj2.get():
+        logger.info("The namespaces openshift-operators-redhat exists,"
+                    "Deleting....")
+        assert namespace_obj2.delete(resource_name='openshift-operators-redhat')
+        logger.info("The project openshift-operators-redhat got"
+                    "deleted successfully")
+    else:
+        logger.error("The project openshift-operators-redhat does not exists")
+
+    # Deletes the catalog source config
+    resources_to_delete = ['cluster-logging-operator', 'elasticsearch']
+    project_obj = ocp.OCP(
+        kind='csc', namespace='openshift-marketplace'
+    )
+    if project_obj.get():
+        logger.info("The CSC resources exists")
+        for resource in range(len(resources_to_delete)):
```

Comment:
There's no need to use the index of the list here, we can simplify this a lot just by iterating over the list naturally.

```python
for resource in resources_to_delete:
    assert project_obj.delete(resource_name=resource)
    logger.info(f"The resource {resource} is deleted")